pkg/sink/kafka/client.go

```diff
@@ -174,18 +173,70 @@ func (p *saramaAsyncProducer) Close() error {
 	return p.producer.Close()
 }
 
-// Input is the input channel for the user to write messages to that they
-// wish to send.
-func (p *saramaAsyncProducer) Input() chan<- *sarama.ProducerMessage {
-	return p.producer.Input()
-}
-
-func (p *saramaAsyncProducer) Successes() <-chan *sarama.ProducerMessage {
-	return p.producer.Successes()
+func (p *saramaAsyncProducer) AsyncCallbackRun(ctx context.Context,
+	changefeedID model.ChangeFeedID,
+	closedChan chan struct{},
```

Comment:
for `changefeed`, `closeChan` and `failpointCh`, can we put them into the `saramaAsyncProducer` struct as fields, to make the signature simple.

Reply:
done